modules/Core/pages/panel/users_edit.php

```diff
@@ -106,6 +106,10 @@
                     Validate::REQUIRED => true,
                     Validate::MIN => 3,
                     Validate::MAX => 20
+                ],
+                'timezone' => [
```

Comment:
Are we sure timezones working this time? So we don't get the same issue we had with registration before as some browsers returned invalid timezones or something idk

Reply:
This won't have the same issue as the one that used to exist during registration, because it does not use JS on the frontend to try and "predict" the timezone; it is just a dropdown select